test/explorer-client.js

```diff
@@ -0,0 +1,41 @@
+import { assert } from 'chai';
+const url = require('url');
+const fetch = require('node-fetch');
+
+import {
+  BlockExplorerClient,
+  BLOCK_EXPLORERS,
+  EXPLORER_SUFFIXES,
+} from '../lib/blockExplorerClient.js';
+
+describe('Block Explorer Client', () => {
+  it('Should instantiate from options', () => {
+
+    const client = BlockExplorerClient.fromOptions({
+      protocol: 'bitcoin',
+      chain: 'main',
+    });
+
+    assert.ok(client);
+  });
+
+  it('Should return transaction urls', () => {
+
+    const chain = 'main';
+    const protocol = 'bitcoin';
+    const txhash = 'foobar';
+
+    const client = BlockExplorerClient.fromOptions({
+      protocol,
+      chain,
+    });
+
+    const links = client.getTransactionLinks(txhash);
+
+    const suffixes = client.getSuffixes();
```

Comment:
no test for `suffixes`?

Reply:
```
  getSuffixes() {
    return this._suffixes;
  }
```

Reply:
`suffixes` looks like it's an unused var though. There are no asserts that the suffix is correct or that this returns what it's expected to.